docs/data-sources/rest_managed.md

```diff
@@ -0,0 +1,41 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# Copied from legacy-docs, in order to make changes change this file in legacy-docs and execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+layout: "aci"
+page_title: "ACI: aci_rest_managed"
+sidebar_current: "docs-aci-data-source-aci_rest_managed"
+subcategory: "Generic"
+description: |-
+  This data source can read one ACI object and its children.
+---
+
+# aci_rest_managed #
+
+This data source can read one ACI object and its children.
+
+## Example Usage ##
+
+```terraform
+data "aci_rest_managed" "example" {
+  dn = "uni/tn-EXAMPLE_TENANT"
+}
+```
+
+## Schema ##
+
+### Required ###
+
+* `dn` - (string) Distinguished Name of the object being managed including its relative name, e.g. uni/tn-EXAMPLE_TENANT.
+
+### Read-Only ###
+
+* `class_name` - (string) Which class object is being created.
+* `id` - (string) The Distinguished Name of the object.
+* `annotation` - (string) Annotation for the class object that is being created. When annotation is provided in content this will take precedence.
```

Comment:
Its not clear from the comment what will take precedence. Annotation in the content or this annotation attribute?

```suggestion
* `annotation` - (string) Annotation for the class object that is being created. When annotation is provided in the content it will take precedence over this annotation attribute.
```

Reply:
changed but slightly different due to functionality change